nikkei/adapter.js

```diff
@@ -0,0 +1,42 @@
+const { Requester, Validator } = require('@chainlink/external-adapter')
+
+const commonKeys = {
+  N225: 'nk225'
+}
+
+const customParams = {
+  base: ['base', 'from', 'asset'],
+  endpoint: false
+}
+
+const createRequest = (input, callback) => {
+  const validator = new Validator(callback, input, customParams)
+  const jobRunID = validator.validated.id
+  const endpoint = validator.validated.data.endpoint || 'get_real_data'
+  const url = `https://indexes.nikkei.co.jp/en/nkave/${endpoint}`
+  let idx = validator.validated.data.base.toUpperCase()
+
+  if (idx in commonKeys) {
```

Comment:
What happens if key is not found? For example, if `base: nk225` is sent, will the result be the same as if `base: n225` is sent as a parameter? 

Former case will send `param.idx: NK225`, and the latter `param.idx: nk225`. I am not sure if API is case sensitive.

Reply:
If key is not found it should just use the input.

The common keys is there so that we can standardize the job spec (Use "N225" on all job specs instead of "nk225" on some and "n225" on others).